### DISCUSSION TITLE
Marking Media as watched in Plex app doesn't reflect in Trakt

### DISCUSSION THREAD

**Discussion by sherbibv** [Taxel/PlexTraktSync#1964] [Support] [answered]
Hello all,
I am not sure since when did this started happening, but if I mark a movie/episode as watched inside Plex app the status in Traks doesn't change. 
If I scrub through the movie till the end the movie will be marked as watched.
Also the progress of the playback is in sync with Trakt, this leads me to belive everything is connected correctly between Plex and Trakt.

Do you guys have any idea what I'm doing wrong? 
Thanks!

**Comment by sherbibv** [comment]
I tried reconfiguring the app with plex and trakt and uppon first setup I get this error (its a screenshot since its easier to read than pasting the code):
![trakt-error](https://github.com/Taxel/PlexTraktSync/assets/10092799/fe4f5bee-d91e-438b-9b97-bfc35c9c8b6a)

Maybe this is an issue? I am not sure.

**Reply by glensc** [reply, marked as answer]
crashes need to be reported as issues. also, screenshots aren't searchable, can't copy them. must report as text.

**Reply by glensc** [reply]
also, one problem one issue/discussion.

**Reply by glensc** [reply]
and your bug report must be detailed as possible, i.e reproducible.
